cornice/cors.py

```diff
@@ -96,8 +96,7 @@ def ensure_origin(service, request, response=None):
                         for o in service.cors_origins_for(method)]):
                 request.errors.add('header', 'Origin',
                                    '%s not allowed' % origin)
-            elif request.headers.get(
-                    'Access-Control-Allow-Credentials', False):
+            elif service.cors_support_credentials_for(method):
```

Comment:
I believe the test is testing for something that in fact should not be true and that's why I eliminated the test.  Let me explain there on your other comment.